lib/optimizely/config/datafile_project_config.rb

```diff
@@ -525,15 +525,17 @@ def generate_feature_variation_map(feature_flags)
       flag_variation_map
     end
 
-    def generate_key_map(array, key)
+    def generate_key_map(array, key, first_value: false)
```

Comment:
first_value seems confusing, please add little bit explanation to get value from 0th index.